inst/_pkgdown.yml

```diff
@@ -44,18 +44,31 @@ home:
    components:
      links:
          title: Links
-         html:
+         text:
              - "The source code is available on Github: <a href=https://github.com/zellerlab/siamcat>https://github.com/zellerlab/siamcat</a>"
              - "SIAMCAT is part of the EMBL microbiome tools</a>. Find out more here: <a href=https://microbiome-tools.embl.de>https://microbiome-tools.embl.de</a>"
              - "<a href=https://microbiome-tools.embl.de> <img src='man/figures/embl_microbiome_tools_logo.png' width=50% height=auto/> </a>"
      funding:
        title: Funding
-       html:
+       text:
            - "We are grateful for funding from the German ministry for education and research (BMBF):"
+           - ""
            - "<a href=https://www.bmbf.de> <img src='man/figures/BMBF.png' width=50% height=auto/> </a>"
+           - ""
            - "SIAMCAT is also part of the de.NBI infrastructure:"
+           - ""
            - "<a href=https://www.denbi.de> <img src='man/figures/denbi.png' width=50% height=auto/> </a>"
 
+footer:
+  right:
+    structure: [privacy]
+    components:
+      privacy: |
```

Comment:
however when I test this locally text is right aligned which is not the case on your live website

Reply:
and link to docs is https://pkgdown.r-lib.org/dev/reference/build_site.html#yaml-config-footer